### ISSUE TITLE
A question about the code

### ISSUE THREAD

**Issue by abedkurdi** [ziyili20/CAMLU#3]
Hello there,
I would like to ask about something in your code: As I know, `fit` function accepts the training data (first argument `x`) and the second argument `y` accepts the labels. What I see in your code is that for both `x` and `y` arguments you used `x_train`. Is there a reason behind this? Am I missing something? The same applies for the `evaluate` function, where you used for `x` and `y` the test set `x_test`.

Thank you in advance.

**Comment by ziyili20** [comment]
Thanks for the question! We use autoencoder here, which is unsupervised and different from the usage in general neural network. For autoencoder, the objective is to minimize the difference between the input x_train and the reconstructed output. Thus both the x and y are the expression values.
